### PR TITLE
Fix command to download results.zip

### LINKED ISSUE #459: Cannot unzip results.zip
In local development with minikube, I'm finding that the `results.zip` that I get using the following command does not unzip:

```bash
kubectl exec -it scheduler-worker-<tab complete here> -- \
  curl --output - "<URL from download button>" > results.zip
```

This command can be [found in the docs here](https://github.com/tweag/chainsail/blob/main/docs/deployment.md#running-the-client-locally).

Here is [an example URL that I used](http://minio.default.svc.cluster.local:9000/chainsail-samples/storage/7/results.zip?AWSAccessKeyId=chainsail&Signature=%2FuTFgCf5ghQIfhPbdcLkuuK4guQ%3D&Expires=1681182729).

Example file: [results.zip](https://github.com/tweag/chainsail/files/11218395/results.zip).

Here's what happens when I tried to unzip:

```sh-session
$ unzip results.zip
Archive:  results.zip
warning [results.zip]:  418 extra bytes at beginning or within zipfile
  (attempting to process anyway)
error [results.zip]:  start of central directory not found;
  zipfile corrupt.
  (please check that you have transferred or created the zipfile in the
  appropriate BINARY mode and that you have compiled UnZip properly)
```

It seems to be a zip file:

```sh-session
$ file results.zip
results.zip: Zip archive data, at least v2.0 to extract, compression method=deflate
```

My version of unzip is 6.00:

```sh-session
$ unzip -V
UnZip 6.00 of 20 April 2009, by Info-ZIP.  Maintained by C. Spieler.  Send
bug reports using http://www.info-zip.org/zip-bug.html; see README for details.

Usage: unzip [-Z] [-opts[modifiers]] file[.zip] [list] [-x xlist] [-d exdir]
  Default action is to extract files in list, except those in xlist, to exdir;
  file[.zip] may be a wildcard.  -Z => ZipInfo mode ("unzip -Z" for usage).

  -p  extract files to pipe, no messages     -l  list files (short format)
  -f  freshen existing files, create none    -t  test compressed archive data
  -u  update files, create if necessary      -z  display archive comment only
  -v  list verbosely/show version info       -T  timestamp archive to latest
  -x  exclude files that follow (in xlist)   -d  extract files into exdir
modifiers:
  -n  never overwrite existing files         -q  quiet mode (-qq => quieter)
  -o  overwrite files WITHOUT prompting      -a  auto-convert any text files
  -j  junk paths (do not make directories)   -aa treat ALL files as text
  -U  use escapes for all non-ASCII Unicode  -UU ignore any Unicode fields
  -C  match filenames case-insensitively     -L  make (some) names lowercase
  -X  restore UID/GID info                   -V  retain VMS version numbers
  -K  keep setuid/setgid/tacky permissions   -M  pipe through "more" pager
See "unzip -hh" or unzip.txt for more help.  Examples:
  unzip data1 -x joe   => extract all files except joe from zipfile data1.zip
  unzip -p foo | more  => send contents of foo.zip via pipe into program more
  unzip -fo foo ReadMe => quietly replace existing ReadMe if archive file newer
```

I found [an article](https://osxdaily.com/2019/05/12/how-to-fix-unzip-error-end-of-central-directory-signature-not-found/) that suggested using `zip -FF` to fix broken zip files:

```sh-session
✗ zip -FF results.zip --out fixed.zip
Fix archive (-FF) - salvage what can
 Found end record (EOCDR) - says expect single disk archive
Scanning for entries...
 copying: /optimization_run0/config.yml  (241 bytes)
 copying: /optimization_run0/dos.pickle  (339 bytes)
 copying: /optimization_run0/energies/energies_replica1_0-500.pickle  (586 bytes)
 copying: /optimization_run0/energies/energies_replica1_500-1000.pickle  (610 bytes)
 copying: /optimization_run0/energies/energies_replica2_0-500.pickle  (620 bytes)
 copying: /optimization_run0/energies/energies_replica2_500-1000.pickle  (622 bytes)
 copying: /optimization_run0/energies/energies_replica3_0-500.pickle  (633 bytes)
 copying: /optimization_run0/energies/energies_replica3_500-1000.pickle  (637 bytes)
 copying: /optimization_run0/energies/energies_replica4_0-500.pickle  (635 bytes)
 copying: /optimization_run0/energies/energies_replica4_500-1000.pickle  (623 bytes)
 copying: /optimization_run0/energies/energies_replica5_0-500.pickle  (601 bytes)
 copying: /optimization_run0/energies/energies_replica5_500-1000.pickle  (602 bytes)
 copying: /optimization_run0/final_stepsizes.pickle  (170 bytes)
 copying: /optimization_run0/samples/samples_replica1_0-500.pickle  (1860 bytes)
 copying: /optimization_run0/samples/samples_replica1_500-1000.pickle  (1925 bytes)
 copying: /optimization_run0/samples/samples_replica2_0-500.pickle  (1970 bytes)
 copying: /optimization_run0/samples/samples_replica2_500-1000.pickle  (2002 bytes)
 copying: /optimization_run0/samples/samples_replica3_0-500.pickle  (2008 bytes)
 copying: /optimization_run0/samples/samples_replica3_500-1000.pickle  (2049 bytes)
 copying:   (526861 bytes)
```

Then I can unzip the `fixed.zip` file with some warnings:

```sh-session
$ unzip ../fixed.zip
Archive:  ../fixed.zip
warning:  filename too long--truncating.
warning:  stripped absolute path spec from /optimization_run0/config.yml
  inflating: optimization_run0/config.yml
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/dos.pickle
  inflating: optimization_run0/dos.pickle   bad CRC 87eb77ab  (should be b8a5f015)
warning:  stripped absolute path spec from /optimization_run0/energies/energies_replica1_0-500.pickle
  inflating: optimization_run0/energies/energies_replica1_0-500.pickle
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/energies/energies_replica1_500-1000.pickle
  inflating: optimization_run0/energies/energies_replica1_500-1000.pickle
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/energies/energies_replica2_0-500.pickle
  inflating: optimization_run0/energies/energies_replica2_0-500.pickle
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/energies/energies_replica2_500-1000.pickle
  inflating: optimization_run0/energies/energies_replica2_500-1000.pickle  (incomplete l-tree)
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/energies/energies_replica3_0-500.pickle
  inflating: optimization_run0/energies/energies_replica3_0-500.pickle
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/energies/energies_replica3_500-1000.pickle
  inflating: optimization_run0/energies/energies_replica3_500-1000.pickle
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/energies/energies_replica4_0-500.pickle
  inflating: optimization_run0/energies/energies_replica4_0-500.pickle
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/energies/energies_replica4_500-1000.pickle
  inflating: optimization_run0/energies/energies_replica4_500-1000.pickle
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/energies/energies_replica5_0-500.pickle
  inflating: optimization_run0/energies/energies_replica5_0-500.pickle
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/energies/energies_replica5_500-1000.pickle
  inflating: optimization_run0/energies/energies_replica5_500-1000.pickle
warning:  stripped absolute path spec from /optimization_run0/final_stepsizes.pickle
  inflating: optimization_run0/final_stepsizes.pickle   bad CRC 1a7b966d  (should be ac3b76e8)
warning:  stripped absolute path spec from /optimization_run0/samples/samples_replica1_0-500.pickle
  inflating: optimization_run0/samples/samples_replica1_0-500.pickle
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/samples/samples_replica1_500-1000.pickle
  inflating: optimization_run0/samples/samples_replica1_500-1000.pickle
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/samples/samples_replica2_0-500.pickle
  inflating: optimization_run0/samples/samples_replica2_0-500.pickle
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/samples/samples_replica2_500-1000.pickle
  inflating: optimization_run0/samples/samples_replica2_500-1000.pickle
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/samples/samples_replica3_0-500.pickle
  inflating: optimization_run0/samples/samples_replica3_0-500.pickle
  error:  invalid compressed data to inflate
warning:  stripped absolute path spec from /optimization_run0/samples/samples_replica3_500-1000.pickle
  inflating: optimization_run0/samples/samples_replica3_500-1000.pickle
  error:  invalid compressed data to inflate
warning:  filename too long--truncating.
mapname:  conversion of  failed
```

### PR DESCRIPTION
**Pull request by steshaw**
The interactive tty wasn't necessary and caused the zip file to be corrupted.

Fixes #459

---

Also switched code snippet syntax highlighting from `console` to `bash` for [slightly better results](https://github.com/tweag/chainsail/blob/b7dce62677f63bcc8f2cc0287a05b6b3b5a2d650/docs/deployment.md). The `console` language is the same as `sh-session`, and expects the shell session to include command prompts and outputs/results. e.g.

With `console`:
```console
$ echo hi
hi
$ echo foo | tr f b
boo
```

With `sh-session`:
```sh-session
$ echo hi
hi
$ echo foo | tr f b
boo
```

However, using `sh-session` is not as convenient to copy-paste because it includes the prompts and outputs. Using `bash` allows you to paste directly into a terminal. e.g.

With `bash`:
```bash
echo hi
echo foo | tr f b
```

### CODE REVIEW

File: docs/deployment.md [hunk 1]
```diff
@@ -95,16 +95,15 @@ With that in hand, you can follow the instruction in the [`frontend README`](app
 In the final call to `yarn run dev`, adapt the URLs in the environment variables to match the output of the service tunnel commands.
 
 :warning: the link provided by the client to download samples won't work when Chainsail is deployed via Minikube, the reason being that the host machine does not see the Minikube-internal DNS server by default. To download sampling results, use the following command:
-```console
-kubectl exec -it scheduler-worker-<tab complete here> -- \
-  curl --output - "<URL from download button>" > results.zip
+```bash
+kubectl exec minio-0 -- curl --output - '<URL from download button>' >results.zip
```
Comment (simeoncarstens) [inline docs/deployment.md:99]: Nice, good to know that `minio-0` also works! :slightly_smiling_face:

**Review by simeoncarstens — APPROVED** [review]
Thanks again for having tracked this down! :slightly_smiling_face: and glad that the fix is so easy. As for the different syntax highlighting, I honestly can't make out a difference, but I'm also somewhat color blind. I fully trust you, though, and will keep that difference in mind! Also TIL about `sh-session`.

File: docs/deployment.md (hunk 1 above)
Comment (steshaw) [reply docs/deployment.md:99]: Yes. I looked for one that didn't rely on tab-completion. I wrote a little script to test this while I was working on the problem and having a fixed pod was easier 😄.

**Comment by steshaw** [comment]
The syntax highlighting the difference is a bit subtle. In larger examples, it's more obvious.

I also learned that [`console` is an alias for `sh-session`](https://github.com/github/linguist/blob/5a0c74277548122267d84283910abd5e0b89380e/lib/linguist/languages.yml#L6386-L6397). I've always used `sh-ession`.

**Pull request merged**